build/codegen/update-generated.sh

```diff
@@ -3,33 +3,40 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -x
 
-bash vendor/k8s.io/code-generator/generate-groups.sh \
+# FIXME: https://issues.redhat.com/browse/OSSM-2367
+CODE_GENERATOR_PACKAGE="k8s.io/code-generator"
+CODE_GENERATOR_VERSION="v0.18.3"
```

Comment:
```suggestion
CODE_GENERATOR_VERSION="$(go list -m -f '{{ if .Replace }} {{ .Replace.Version }} {{ else }} {{ .Version }} {{ end }}' ${CODE_GENERATOR_PACKAGE})"
```

This will pull the module version from `go.mod`. If you use this you can probably remove the FIXME above :)

Reply:
Thanks a lot for this!!!!!